src/upgrade/upgrade_scripts/5.21.0/remove_datachunks_index.js

```diff
@@ -0,0 +1,21 @@
+/* Copyright (C) 2025 NooBaa */
+"use strict";
+
+async function run({ dbg, db_client }) {
+  const indexName = 'idx_btree_datachunks_dedup_key';
+
+  try {
+    const pool = db_client.instance().get_pool();
+    await pool.query(`DROP INDEX IF EXISTS ${indexName};`);
+
+    dbg.log2("Executed upgrade script for dropping index ", indexName);
```

Comment:
dbg.log2 will never be printed as the default is dbg0, and it is an upgrade script. 
```suggestion
    dbg.log0("Executed upgrade script for dropping index ", indexName);
```